docs/manage/fleet/system-settings.md

```diff
@@ -13,76 +13,33 @@ The `viam-agent` configuration allows you to configure:
 
-## Configure OS package updates
+## Manage OS package updates
 
 By default, the configuration in <FILE>/etc/apt/apt.conf.d/</FILE> determines the behavior for updating operating system packages.
-To adjust these settings update the `"agent"` value in the machine's JSON configuration.
-
-For complete reference information, see [viam-agent](/configure/agent/#agent-syscfg).
-
-### Automatically upgrade all packages
-
-When set, `viam-agent` will install the `unattended-upgrades` package and replace `20auto-upgrades` and `50unattended-upgrades` in <FILE>/etc/apt/apt.conf.d/</FILE> with an Origins-Pattern list generated automatically from configured repositories on the system.
-Custom repos installed on the system at the time the setting is enabled will be included.
-
-To set automatic upgrades for all packages, add the `upgrades` field to the `agent-syscfg`'s `attributes` object as following.
-You may need to add the `agent-syscfg` object to the `agent` object if it doesn't already exist.
-
-
-```json
-"agent": {
-    "agent-syscfg": {
-        "release_channel": "stable",
-        "attributes": {
-            "upgrades": {
-                "type": "all"
-            }
-        }
-    }
-}
-```
-
-### Automatically upgrade security packages
-
-When set, `viam-agent` will install the `unattended-upgrades` package and replace `20auto-upgrades` and `50unattended-upgrades` in <FILE>/etc/apt/apt.conf.d/</FILE> with an Origins-Pattern list generated automatically from configured repositories on the system.
-Custom repos installed on the system at the time the setting is enabled will be included.
-
-To set automatic upgrades for packages containing `"security"` in their codename (for example `bookworm-security`), add the `upgrades` field to the `agent-syscfg`'s `attributes` object as following.
-You may need to add the `agent-syscfg` object to the `agent` object if it doesn't already exist.
+To manage OS package updates using Viam, add an `"agent-syscfg"` object to the `"agent"` object in the machine's JSON configuration, if it doesn't already exist.
+Then, add the `"upgrades"` field in its attributes:
 
 ```json
 "agent": {
     "agent-syscfg": {
         "release_channel": "stable",
         "attributes": {
             "upgrades": {
-                "type": "security"
+                "type": "all|security|disabled"
             }
         }
     }
 }
 ```
 
-### Disable automatic upgrades
-
-When set, `viam-agent` will install the `unattended-upgrades` package and replace `20auto-upgrades` and `50unattended-upgrades` in <FILE>/etc/apt/apt.conf.d/</FILE> with an Origins-Pattern list generated automatically from configured repositories on the system.
+When the `upgrades` attribute is specified, `viam-agent` will install the `unattended-upgrades` package and replace `20auto-upgrades` and `50unattended-upgrades` in <FILE>/etc/apt/apt.conf.d/</FILE> with an Origins-Pattern list generated automatically from configured repositories on the system.
```

Comment:
```suggestion
When the `type` attribute is set to `upgrades`, `viam-agent` will install the `unattended-upgrades` package and replace `20auto-upgrades` and `50unattended-upgrades` in <FILE>/etc/apt/apt.conf.d/</FILE> with an Origins-Pattern list generated automatically from configured repositories on the system.
```
[nit] `type` is the attribute. `upgrades` is the value of that attribute.

Reply:
what you're describing would mean `"type": "upgrades"`. I can say the type field is set for `"upgrades"`.

```suggestion
When the `type` attribute is specified for `"upgrades"`, `viam-agent` will install the `unattended-upgrades` package and replace `20auto-upgrades` and `50unattended-upgrades` in <FILE>/etc/apt/apt.conf.d/</FILE> with an Origins-Pattern list generated automatically from configured repositories on the system.
```